src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/Processors/OperationFilteringTelemetryProcessor.cs

```diff
@@ -18,6 +21,24 @@ public OperationFilteringTelemetryProcessor(ITelemetryProcessor next)
 
         public void Process(ITelemetry item)
         {
+            // There are scenarios in Functions where we can have two TelemetryClients running for a short period.
+            // We use this property to ensure we only send one of the RequestTelemetry objects (auto-tracked by App 
+            // Insights) to the server.
+            if (item is RequestTelemetry request)
+            {
+                lock (_lock)
+                {
+                    if (request.Properties.ContainsKey(ProcessedKey))
```

Comment:
nit: can we simplify this by setting if the key is not present instead (eliminate the `else`)

Reply:
If it's there, we return -- thereby short-circuiting the telemetry (it won't be uploaded). Note we don't call `_next.Process()` in that case. So if it's there, exit.

Reply:
Sorry... had an issue with my diff (same reason for the question below), the else can still be removed, though :)